webservice/utils/get-data-from-form-section.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var assign = require('es5-ext/object/assign');
+
+module.exports = function getDataFromSection(dataForm, result, typeMapper) {
+
+	if (!result) result = {};
+
+	var ownerType = dataForm.propertyMaster;
+
+	// first try toWsSchemaJson method
+	if (dataForm.toWebServiceSchemaJSON) {
+		var dataFormResult = dataForm.toWebServiceSchemaJSON(typeMapper);
+		return assign(result, dataFormResult);
+	}
+
+	// if tabular section
+	if (dataForm.propertyName) {
```

Comment:
This is some serious duck typing, not really safe. Better check for instance of `FormEntitiesTable`. Still I don't see why `FormEntitiesTable` can't have `toWebServiceSchemaJSON`?